carsus/io/output/base.py

```diff
@@ -678,8 +667,17 @@ def create_cross_sections(self):
         """
 
         logger.info('Ingesting photoionization cross-sections.')
+        cross_sections_reader = self.cmfgen_reader.cross_sections
+
+        # remove cross-sections above the ionization threshold
+        phixs_clean = []
+        for ion in self.cmfgen_ions:
+            ion_phixs = cross_sections_reader.loc[ion]
+            threshold = self.ionization_energies.base[ion]
+            ion_phixs = ion_phixs[ ion_phixs['energy'] < threshold ]
```

Comment:
Does this not also remove the high energy part of the cross section table for levels below the ionization threshold?
Say we have an ionization threshold of 10 eV and a level with an excitation energy of 1 eV; In this case the table would start at 9 eV and then be incorrectly truncated at 10 eV?

Reply:
On second thought, I don't think this removes anything that we want to be remove.

Reply:
Yes, it's clear for me now